src/buttons/flat-button-label.jsx

```diff
@@ -61,19 +71,16 @@ const FlatButtonLabel = React.createClass({
       style,
     } = this.props;
 
-    const contextKeys = this.constructor.getRelevantContextKeys(this.state.muiTheme);
+    const {
+      prepareStyles,
+    } = this.state.muiTheme;
 
-    const mergedRootStyles = this.mergeStyles({
-      position: 'relative',
-      paddingLeft: contextKeys.spacingDesktopGutterLess,
-      paddingRight: contextKeys.spacingDesktopGutterLess,
-    }, style);
+    const styles = getStyles(this.props, this.state);
 
     return (
-      <span style={this.prepareStyles(mergedRootStyles)}>{label}</span>
+      <span style={prepareStyles(styles.root, style)}>{label}</span>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/344018/12865318/a70e1178-cc76-11e5-91c3-240316a2d080.png)

:laughing: :laughing: The docs site got very angry with me.